presto-docs/src/main/sphinx/release/release-329.rst

```diff
@@ -0,0 +1,58 @@
+=========================
+Release 329 (XXX 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect result of ``last_day_of_month`` function for first day of month. (:issue:`2452`)
```

Comment:
Link to function and say "for" instead of "of"
```
* Fix incorrect result for :func:`last_day_of_month` function for first day of month. (:issue:`2452`)